services/data-info/src/data_info/routes/stats.clj

```diff
@@ -9,6 +9,8 @@
 
 (defroutes* stat-gatherer
 
+            ; FIXME Update metadactyl exception handling when data-info excptn hndlg updated
+            ; metadactyl catches exceptions thrown from this EP.
```

Comment:
Noting of course that this is called through Donkey as well, so probably that'll need updating at the same time too? Not sure.